### PR TITLE
RFrame - introduce RTitle class

### PR DESCRIPTION
**Pull request by linev**
It is fully bound to RFrame and only margin and height can be specified.
In the code one does:

     auto title = subpads[1][0]->Draw<RTitle>("Frame2 title");
     title->SetMargin(0.01_normal).SetHeight(0.05_normal);

Provide preliminary JSROOT code.

Also first try to provide context menu - for `RAttrAxis` 
Now only lin/log scale switch there - only prove of principle

Also introduce `kNone` attribute kind in RAttrMap. 
Allows to suppress values which is configured in RStyle or in the parent classes.

Introduce `zoommin`/`zoommax` values for `RAttrAxis`.

### CODE REVIEW

File: graf2d/primitivesv7/inc/ROOT/RTitle.hxx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+/** \class ROOT::Experimental::RTitle
+\ingroup GrafROOT7
+\brief A title for the RFrame.
+\author Sergey Linev <s.linev@gsi.de>
+\date 2020-02-26
+\warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback is welcome!
+*/
+
+
+class RTitle : public RDrawable {
```
Comment (Axel-Naumann) [inline graf2d/primitivesv7/inc/ROOT/RTitle.hxx]: `RFrameTitle`? "Title" could be anything. Or is it meant to also be the axis title, the canvas title, etc? I guess not...

File: tutorials/v7/draw_frame.cxx [hunk 2]
```diff
@@ -32,16 +34,16 @@ auto style = std::make_shared<RStyle>(); // keep here to avoid destroy when leav
 void draw_frame()
 {
    // Create the histogram.
-   RAxisConfig xaxis("x", 10, 0., 1.);
+   RAxisConfig xaxis("x", 10, 0.1, 110);
```
Comment (Axel-Naumann) [inline tutorials/v7/draw_frame.cxx]:
```suggestion
   RAxisConfig xaxis("x", 10, 0.1, 110.);
```

File: graf2d/gpadv7/src/RAttrBase.cxx [hunk 3]
```diff
@@ -92,37 +95,64 @@ void ROOT::Experimental::RAttrBase::AssignParent(RAttrBase *parent, const std::s
...
 void ROOT::Experimental::RAttrBase::ClearValue(const std::string &name)
 {
    if (auto access = AccessAttr(name))
-       const_cast<RAttrMap*>(access.attr)->Clear(access.fullname);
+       access.attr->Clear(access.fullname);
+}
+
+///////////////////////////////////////////////////////////////////////////////
+/// Add "None" value to attribute. Ensure that value can not be configured via style
+/// Equivalent to css syntax { attrname: }
```
[the remark this answers is not in the document]
Comment (linev) [reply graf2d/gpadv7/src/RAttrBase.cxx]: Yes, probably `NoValue` is better name. 

But would not use `ResetValue` or `ClearValue` as functions name. 
Function name should say - set `NoValue` for attribute. 
I will adjust it.

File: graf2d/primitivesv7/inc/ROOT/RTitle.hxx (hunk 1 above)
Comment (linev) [reply graf2d/primitivesv7/inc/ROOT/RTitle.hxx]: For the moment it is not that important - I can rename it to `RFrameTitle`.
I just want to introduce class which does not use absolute position, but relative to something else. And test complete tool chain - object, attributes, display in JS, interactivity, ....
More interesting are attributes for `RTitle` - they can be reused for different components. Like axis title.

**Pull request merged**